Package.swift

```diff
@@ -65,6 +65,7 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-macro-testing", from: "0.4.0"),
         .package(url: "https://github.com/pointfreeco/swift-perception", from: "1.5.0"),
         .package(url: "https://github.com/pointfreeco/swift-custom-dump", from: "1.2.1"),
+        .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay", from: "1.0.0"),
```

Comment:
we currently already depend on this transitively through 'swift-custom-dump'. added as an explicit dependency because we're directly importing it in some spots now, and i assume that's the 'right thing to do'.